### ISSUE TITLE
Gnome Builder really needs cutting edge deps?

### ISSUE THREAD

**Issue by kirushyk** [chergert/gnome-builder#55]
-m4_define([gtk_required_version], [3.19.1])
    -m4_define([glib_required_version], [2.47.1])
    -m4_define([gtksourceview_required_version], [3.19.0])

etc.

3.18.x will not work?

**Comment by chergert** [comment]
Yes, Builder is built alongside those upstream libraries and we regular commit features/fixes we need to them.
